internal/server/run.go

```diff
@@ -30,17 +29,15 @@ func run(c config.Config) (err error) {
 
 	// configure cache dir
 	utils.SetCacheDir(c.CacheDir)
-	log.Logger.Debugf("cache dir:  %s", utils.CacheDir())
-
-	fsCache, err := cache.NewFSCache(utils.CacheDir())
+	cache, err := operation.NewCache(c.CacheBackend)
 	if err != nil {
-		return xerrors.Errorf("unable to initialize cache: %w", err)
+		return xerrors.Errorf("server cache error: %w", err)
 	}
+	defer cache.Close()
+	log.Logger.Debugf("cache dir:  %s", utils.CacheDir())
```

Comment:
Do you want to print it even if the cache is resolved to the Redis backend?

Reply:
Yes, as I mentioned in https://github.com/aquasecurity/trivy/pull/770#discussion_r541842537, the vulnerability database still depends on this cache dir.